### DISCUSSION TITLE
Preprocessing BRENDA data

### DISCUSSION THREAD

**Discussion by feiranl** [SysBioChalmers/GECKO#177] [GECKO3]
Hi,

The original GECKO pipeline relies on full substrate name matching, which will has some issues with ID mapping.
Since we are thinking of having SMILES in the model (search by PubChem API), if we can also preprocess BRENDA to store the substrates SMILEs in the file, then it would facilitate the matching process and the kcat searching step.

**Comment by edkerk** [comment]
Good idea, is this already gathered for DLKcat?

**Reply by feiranl** [reply]
Yes.

**Reply by edkerk** [reply]
Great, it would anyway be easiest if we just parse that file to also use it in DLKcat (#157).
